modules/terraform/cmd.go

```diff
@@ -77,10 +90,19 @@ func GetExitCodeForTerraformCommand(t *testing.T, additionalOptions *Options, ar
 // GetExitCodeForTerraformCommandE runs terraform with the given arguments and options and returns exit code
 func GetExitCodeForTerraformCommandE(t *testing.T, additionalOptions *Options, additionalArgs ...string) (int, error) {
 	options, args := GetCommonOptions(additionalOptions, additionalArgs...)
+	binary := additionalOptions.TerraformBinary
+
+	if binary == "" {
+		binary = "terraform"
+	}
+
+	if binary == "terragrunt" {
+		args = append(args, "--terragrunt-non-interactive")
+	}
```

Comment:
Since `GetCommonOptions` has `additionalOptions`, can this logic be added there instead?